pkg/hierarchy.go

```diff
@@ -16,12 +18,13 @@ type Config struct {
 }
 
 type Item struct {
-	Kind      string       `yaml:"kind" json:"kind"`
-	Name      string       `yaml:"name" json:"name"`
-	Namespace string       `yaml:"namespace,omitempty" json:"namespace,omitempty"`
-	Type      ResourceType `yaml:"type,omitempty" json:"type,omitempty"`
-	Key       string       `yaml:"key,omitempty" json:"key,omitempty"`
-	Index     int          `yaml:"index,omitempty" json:"index,omitempty"`
```

Comment:
Can you add the yaml and json tags back please, they are required fro creating CRDs and other auto-generated files by operator

Reply:
Will do.

Reply:
Changed